packages/@n8n/nodes-langchain/nodes/vendors/GoogleGemini/helpers/utils.ts

```diff
@@ -165,3 +225,100 @@ export async function transferFile(
 
 	return { fileUri: file.uri, mimeType: file.mimeType };
 }
+
+export async function createFileSearchStore(
+	this: IExecuteFunctions,
+	displayName: string,
+): Promise<IDataObject> {
```

Comment:
Response type declaration should not be needed here since the response is casted as a specific type.